File: cpp/useHelper.cpp
#include <iostream>
#include <string>

void useHelper(std::string selector, std::string word) {
    std::string result;

    if (selector == "js-helper-show-letter-text") {
        result = word[0];
        result[0] = std::toupper(result[0]);
    } else if (selector == "js-helper-show-letters-length-text") {
        result = std::to_string(word.length());
    } else if (selector == "js-helper-show-full-word-text") {
        result = word;
        for (char &c : result) {
            c = std::toupper(c);
        }
    } else {
        std::cout << "Неверный тип изменения" << std::endl;
        return;
    }

    std::cout << result << std::endl;
}

File: js/functions/helpers/useHelper.js
function useHelper(selector, word) {
    const field = document.querySelector(`.${selector}`);
    let result = null;

    switch (selector) {
        case 'js-helper-show-letter-text':
            result = word[0].toUpperCase();
            break;
        case 'js-helper-show-letters-length-text':
            result = word.length;
            break;
        case 'js-helper-show-full-word-text':
            result = word.toUpperCase();
            break;
        default:
            console.log('Неверный тип изменения');
    }

    field.innerHTML = result;
    field.classList.add('active');
    setTimeout(()=> field.classList.remove('active'), 3000)
}

export default useHelper;